lib/lightning/impact_tracking/response_processor.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Lightning.ImpactTracking.ResponseProcessor do
```

Comment:
Pulled this into a module of its own (rather than exposing the functionality as a public method on `ImapctTracking.Client`) to make it easier to test directly.